File: fortran/triangulo.f90
program DesenhaTriangulo
    implicit none
    integer :: altura, i, j

    altura = 20

    do i = 1, altura
        ! Espaços à esquerda
        do j = 1, altura - i
            write(*, '(A)', advance='no') ' '
        end do

        ! Caracteres no centro
        do j = 1, 2 * i - 1
            write(*, '(A)', advance='no') '#'
        end do

        ! Nova linha após cada linha do triângulo
        write(*, '(A)') ''
    end do

end program DesenhaTriangulo

File: rust/triangulo.rs
fn main() {
    let altura = 20;

    for i in 1..=altura {
        // Espaços à esquerda
        for _ in 0..(altura - i) {
            print!(" ");
        }

        // Caracteres no centro
        for _ in 0..(2 * i - 1) {
            print!("#");
        }

        // Nova linha após cada linha do triângulo
        println!();
    }
}
